frontend-graphql/src/components/Footer.js

```diff
@@ -1,30 +1,12 @@
 import React from 'react';
+import { ReactComponent as Logo } from '../static/images/postlight-logo.svg';
```

Comment:
I am getting an error here that is crashing the GraphQL frontend:
```
Module not found: Can't resolve '../static/images/postlight-logo.svg' in '/home/node/app/src/components'
```

Reply:
fixed by https://github.com/postlight/headless-wp-starter/pull/244/commits/e13d6fd528739e6268458318562adfaeed379339